packages/ingest/src/steps/2-lp-batch-start.ts

```diff
@@ -53,7 +56,8 @@ export async function lpBatchStart({
       },
     });
   } catch (error) {
-    console.error("Error generating lesson plans", error);
+    log.error(String(error));
+    log.info("Error starting lesson plan generation, see error above");
```

Comment:
I think this should also be an error, as I assume we would want to show it even if you've opted out of info logs for this namespace